source/optional_features/corporate_card_surcharges/index.html.md.erb

```diff
@@ -0,0 +1,41 @@
+---
+title: Corporate card surcharges 
+weight: 134
+---
+
+# Corporate card surcharges  
+
+GOV.UK Pay supports adding a surcharge when a user pays with a corporate
+credit or debit card. You can [contact GOV.UK Pay
+support](/support_contact_and_more_information/#contact-us) to ask the team to
+set this up for you. 
+
+Each surcharge is a flat amount added to a payment, not a percentage. 
+
+Different corporate surcharges are possible for 4 different types of card.
+These are: 
+
+* non-prepaid credit cards 
+* non-prepaid debit cards
+* corporate pre-paid credit cards
+* corporate pre-paid debit cards 
+
+GOV.UK Pay maintains a list of cards of all 4 types. This means the platform
+can identify when a user enters a card number that matches one of these. If a
+surcharge is applicable, GOV.UK Pay will inform the user that a corporate
+surcharge of a certain amount will be added to the total. If the platform
+cannot determine the exact type of a card being used, it will not apply a
+surcharge.
+
+You can sign in to the [GOV.UK Pay admin
+tool](https://selfservice.payments.service.gov.uk/login) to see the
+payment amount and the surcharge amount listed separately. 
+
+If a corporate surcharge is applied to a particular payment, you can see this
+in API responses. The `"corporate_card_surcharge"` and `"total _amount"`
+fields contain amounts in pence. 
+
+If you issue a refund, you can optionally refund up to the full amount -
+including the surcharge. You can also use the partial refund feature to only
```

Comment:
I’m not sure “optionally refund up to the full amount” gets across that this is the default behaviour and I think we want to make sure we get across that the `amount_available` and `refund_amount_available` fields in the API also include the surcharge.

Maybe something like this? “If a payment had a surcharge applied, the full amount available for refund includes the surcharge. You can also use the partial refund feature to only …”

Maybe we should also update the refunding payments page to indicate that the `amount_available` and `refund_amount_available` fields include any surcharge.